modules/manage/pages/cluster-maintenance/compaction-settings.adoc

```diff
@@ -26,6 +26,56 @@ Compaction policy may be applied to a cluster or to an individual topic. If both
 * `compact`: This triggers only cleanup of messages with multiple versions. A message that represents the only version for a given key is not deleted.
 * `compact,delete`: This combines both policies, deleting messages exceeding the retention period while compacting multiple versions of messages.
 
+== Tombstone record removal
+
+When a topic's messages should be deleted, for example as data is deleted from a source system, clients add a tombstone record to the message log. A tombstone contains the message key, and the value `null`. Tombstones signal to brokers and consumers that records with the same key prior to it in the log should be deleted. 
+
+You can specify how long Redpanda keeps these tombstones for compacted topics using both a cluster configuration property `tombstone_retention_ms` and a topic configuration property xref:reference:properties/topic-properties.adoc#deleteretentionms[`delete.retention.ms`]. If both are set, the topic-level tombstone retention policy overrides the cluster-level policy.
+
+[NOTE]
+====
+Redpanda does not currently remove tombstone records for compacted topics with Tiered Storage enabled. 
+
+You cannot enable `tombstone_retention_ms` if you have enabled any of the Tiered Storage cluster properties `cloud_storage_enabled`, `cloud_storage_enable_remote_read`, and `cloud_storage_enable_remote_write`.
+
+On the topic level, you cannot enable `delete.retention.ms` at the same time as the Tiered Storage topic configuration properties `redpanda.remote.read` and `redpanda.remote.write`.
+====
+
+To set the cluster-level tombstone retention policy, run the command:
+
+[,bash]
+----
+rpk cluster config set tombstone_retention_ms=100
+----
+
+You can unset the tombstone retention policy for a topic so it inherits the cluster-wide default policy:
+
+[,bash]
+----
+rpk topic alter-config <topic-name> --delete delete.retention.ms
+----
+
+To override the cluster-wide default for a specific topic:
+
+[,bash]
+----
+rpk topic alter-config <topic-name> --set delete.retention.ms=5
+----
+
+To disable tombstone removal for a specific topic:
+
+[,bash]
+----
+rpk topic alter-config <topic-name> --set delete.retention.ms=-1
+----
+
+Redpanda removes tombstones as follows:
+
+* For topics with a `compact,delete` cleanup policy: the standard garbage collection process, per the topic retention limit.
```

Comment:
@WillemKauf just to double check, it's the topic retention limit here and not tombstone?

Reply:
The point we are trying to communicate here is that in `compact,delete` configured topics, tombstones can be removed by the compaction process per the tombstone limits (as we have described in the `compact` process with `delete.retention.ms`), but standard garbage collection would also apply to the topic (e.g if the log were to be garbage collected up to an offset 10, all records with offsets < 10 would be removed, including tombstones records in that range).